template/src/pageComponents/buy-me-coffee/FormBuyCoffee.tsx

```diff
@@ -2,13 +2,23 @@ import React, { useState, useCallback, useMemo } from 'react';
 
 import clsx from 'clsx';
 import { parseEther } from 'viem';
-import { useContractWrite, usePrepareContractWrite, useWaitForTransaction } from 'wagmi';
+import {
+  useAccount,
+  useContractWrite,
+  usePrepareContractWrite,
+  useWaitForTransaction,
+} from 'wagmi';
 import { useBuyMeACoffeeContract } from '../../hooks/contracts';
+import { useBalance } from 'wagmi';
+import { useLoggedInUserCanAfford } from '../../hooks/useUserCanAfford';
 
 type FormBuyCoffeeProps = {
   onComplete: () => void;
 };
 
+const BUY_COFFEE_AMOUNT_RAW = '0.5';
```

Comment:
That's an expensive coffee LOL.

I believe the coffee we now is `0.0001`, so `0.0002` should be enough.

Reply:
LOL good catch. Just reverted to 0.0001. We'll revisit the price in a couple weeks based on value added 😂